packages/pluggableWidgets/column-chart-web/src/ColumnChart.xml

```diff
@@ -136,45 +115,78 @@
                                     <enumerationValue key="last">Last</enumerationValue>
                                 </enumerationValues>
                             </property>
-                            <property key="customSeriesOptions" type="string" required="false" multiline="true">
-                                <caption>Custom series options</caption>
-                                <description/>
+                            <property key="staticTooltipHoverText" type="textTemplate" required="false" multiline="false" dataSource="staticDataSource">
+                                <caption>Tooltip hover text</caption>
+                                <description />
+                            </property>
+                            <property key="dynamicTooltipHoverText" type="textTemplate" required="false" multiline="false" dataSource="dynamicDataSource">
+                                <caption>Tooltip hover text</caption>
+                                <description />
                             </property>
                         </propertyGroup>
                         <propertyGroup caption="Appearance">
                             <property key="barColor" type="textTemplate" required="false" multiline="false">
-                                <caption>Bar color</caption>
+                                <caption>Column color</caption>
                                 <description>Color of the column e.g. green, #00FF00, rgb(0,255,0)</description>
                             </property>
                         </propertyGroup>
+                        <propertyGroup caption="Advanced">
+                            <property key="customSeriesOptions" type="string" required="false" multiline="true">
+                                <caption>Custom series options</caption>
+                                <description />
+                            </property>
+                        </propertyGroup>
```

Comment:
We normally put the Advanced group as the last in the group, so in this case after propertyGroup "Events". Can you change it for consistency with other charts? ([example](https://github.com/mendix/widgets-resources/pull/1350/commits/87f589113b85780aabddb07f395bc03fe819e0cc#diff-41ea85ddc15d7b5eb3f83cf1f9d33429a867f18af45fe5e3b268797ebd59ab3cR139))